scripts/unsupported/combine_tracks_postprocessing_cnv/combine_tracks.wdl

```diff
@@ -0,0 +1,621 @@
+			MergeAnnotatedRegionsByAnnotation \
+            --segments ${seg_file} \
+            ${"--max-merge-distance " + max_merge_distance} \
+            --annotations-to-match ${sep=" --annotations-to-match " annotations} \
+            -O ${output_name}.merged.seg -R ${ref_fasta}
+
+	>>>
+
+    runtime {
+        docker: gatk_docker
+        memory: machine_mem + " MB"
+        # Note that the space before SSD and HDD should be included.
+        disks: "local-disk " + select_first([disk_space_gb, default_disk_space_gb]) + if use_ssd then " SSD" else " HDD"
+        preemptible: select_first([preemptible_attempts, 3])
+        cpu: select_first([cpu, 1])
+        bootDiskSizeGb: 20
+    }
+
+    output {
+        File cnv_merged_seg = "${output_name}.merged.seg"
+    }
+}
+
+# TODO: No non-trivial heredocs in WDL.  Add this to the script directory and call via anaconda (future release)
+# TODO: This is a ham-fisted algorithm.  Better approaches exist if this does not meet needs.
+task PrototypeACSConversion {
+    File model_seg
+    File af_param
+    Float? maf90_threshold
+    String output_filename = basename(model_seg) + ".acs.seg"
+    String output_skew_filename = output_filename + ".skew"
+
+    command <<<
+        set -e
+        python <<EOF
+import sys
+import re
+import pandas as pd
+import numpy as np
+from collections import defaultdict
+import scipy
+from scipy import special as sp
+import os.path
+
+model_segments_seg_input_file = "${model_seg}"
+model_segments_af_param_input_file = "${af_param}"
+alleliccapseg_seg_output_file = "${output_filename}"
+alleliccapseg_skew_output_file = "${output_skew_filename}"
+
+HAM_FIST_THRESHOLD=${default="0.485" maf90_threshold}
+
+# regular expression for matching sample name from header comment line
+sample_name_header_regexp = "^@RG.*SM:(.*)[\t]*.*$"
+
+#define AllelicCapSeg columns
+alleliccapseg_seg_columns = [
+    'Chromosome',
+    'Start.bp',
+    'End.bp',
+    'n_probes',
+    'length',
+    'n_hets',
+    'f',
+    'tau',
+    'sigma.tau',
+    'mu.minor',
+    'sigma.minor',
+    'mu.major',
+    'sigma.major',
+    'SegLabelCNLOH']
+
+def read_sample_name(input_file, max_scan_lines=10000):
+    with open(input_file, 'r') as f:
+        for _ in range(max_scan_lines):
+            line = f.readline()
+            match = re.search(sample_name_header_regexp, line, re.M)
+            if match is None:
+                continue
+            groups = match.groups()
+            return groups[0]
+    raise Exception("Sample name could not be found in \"{0}\"".format(input_file))
+
+#read GATK ModelSegments files and perform some basic checks
+model_segments_seg_pd = pd.read_csv(model_segments_seg_input_file,
+                                    sep='\t', comment='@', na_values='NA')
+model_segments_af_param_pd = pd.read_csv(model_segments_af_param_input_file, sep='\t', comment='@')
+
+def simple_determine_allelic_fraction(model_segments_seg_pd):
```

Comment:
What happened to the beta fitting?

Reply:
I found that it did very little on the test data that I had on hand.  I do not yet have a good evaluation set up for the ABSOLUTE conversion, which I would like to do before making more changes.  We can discuss offline -- I think I have an approach.

No action.